pyproject.toml

```diff
@@ -26,11 +26,12 @@ factory-boy = "^3.3.3"
 optional = true
 
 [tool.poetry.group.dev.dependencies]
+black = "^25.1.0"
 factory-boy = "^3.3.3"
+isort = "^6.0.1"
```

Comment:
To support running checks locally, so that we can verify things without waiting for Github actions to run.